src/_shared/scss/_popup.scss

```diff
@@ -0,0 +1,297 @@
+/* ==========================================================================
```

Comment:
Looks like there was a single useful property being taken from the entire file, so I copied it to index and dropped the file.